pupil-spa/src/app/check/check.component.ts

```diff
@@ -48,11 +49,15 @@ export class CheckComponent implements OnInit {
 
   manualSubmitHandler(answer: string) {
     // console.log(`check.component: manualSubmitHandler(): ${answer}`);
+    const answerSet = { factor1: this.question.factor1, factor2: this.question.factor2, answer: parseInt(answer, 10) };
```

Comment:
@jon-shipley is that required? I would assume you will always have the answers in the proper order, hence you will be able to distinguish the sequence.